libc-test/build.rs

```diff
@@ -2002,6 +2002,8 @@ fn test_freebsd(target: &str) {
             // not available until FreeBSD 12, and is an anonymous union there.
             ("xucred", "cr_pid__c_anonymous_union") => true,
 
+            ("umutex", "m_owner") => true,
```

Comment:
Could you add a comment describing why this is skipped? It helps us understand when cleaning up tests.